server/src/main/java/io/unitycatalog/server/auth/JCasbinAuthenticator.java

```diff
@@ -0,0 +1,60 @@
+package io.unitycatalog.server.auth;
+
+import io.unitycatalog.server.model.Privilege;
+import org.casbin.jcasbin.main.Enforcer;
+
+import java.util.List;
+import java.util.UUID;
+
+public class JCasbinAuthenticator implements UnityCatalogAuthenticator {
+    Enforcer enforcer = new Enforcer("src/main/resources/jcasbin_auth_model.conf");
```

Comment:
Can Enforcer take some sort of `Stream` like from `Class.resourceAsStream()` to find this in the classpath?

Reply:
I could read it into a string first and then create a `Model` object and then pass that model object to `Enforcer` but that seemed a bit more complex that it needs to be.

Reply:
I _think_ we are going to need to do something because in the packaged code won't have the file in `src/main/resources/` folder. I will probably just be at the root in the JAR, but we can investigate later.